types/consumer-data-standards/banking/index.d.ts

```diff
@@ -8201,6 +8217,10 @@ export interface ResponseBankingTransactionList {
          */
         totalRecords: number;
         [k: string]: unknown;
+        /**
+         * true if "text" query parameter is not supported
+         */
+        isQueryParamUnsupported?: boolean | null;
```

Comment:
Please move so that `[k: string]: unknown;` is the last member of the interface.